metadata_annotation_gdsc.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>Is there a way this could be just two lines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>